proposals/demonstration-organisation.md

```diff
@@ -0,0 +1,157 @@
+---
+
+**Checklist**
+
+
+<!--> Add extra fields for management here <!-->
+
+- [x] Template passes validation with `trestle`
+- [x] Proposal cuts across multiple `oscal-compass` projects
+
+
+
+## Summary/Abstract
+
+OSCAL compass is built on the premise of compliance as code and gitops.
+This means that Github repositories, and organisations, are used for demonstrations including integration with CICD platforms such as github actions.
+The result is that curating this content is key.
+This proposal outlines guidelines and processes for managing demonstration content in a consistent and coherent way.
+
+## Background
+
+Over time there have been a number of demonstrations built for OSCAL compass.
+Many of these demonstrations contain overlapping functionality, however, clear ownership has not been maintained.
+Most of these demonstrations have not been reviewed at the community level, whether due to historical creation or lack of process.
+The result is we do not have a consistent set of project demonstration content.
+
+### Motivation and problem space
+
+- To improve the demonstration content itself.
+- To ensure that the primary github organization (e.g. https://github.com/oscal-compass) is a focused and clean as possible. 
+  - This will enable the 'core' to meet standards such as OpenSSF more easily
+- To ensure a minimum viable set of processes in place to manage the demo content.
+
+
+### Impact and desired outcome
+
+- Avoiding creating a dumping zone: Organizations which have 100's or 1000's of garbage projects which have been thrown into the wild.
+- W
+
+### Prior discussion and links
+
+- Mostly via slack / f2f conversations.
+
+
+## User/User Story (Optional)
+
+- As an Oscal compass contributor I have a clear process for obtaining and creating 'official' demo repositories
+- As an Oscal compass user I have a single place for clear and consistent demonstration content
+- As a potential user of Oscal compass the core organization clearly shows the projects 
+
+## Goals
+
+- Use this *inception* proposal to define a simple template
+- Enforce the use of this template by using `trestle` and a GitHub Actions pipeline
+- Extend the documentation in the [proposals README.md](./README.md) to cover the use of trestle for proposals
+- Minimize the authors effort by using GitHub metadata where appropriate.
+
+## Non-Goals
+
+- Demonstrations are not the default for regression tests. Testing should be part of the main project.
+- To decide what are the priority demonstrations for oscal-compass
+
+## Proposal
+
+1. Reference demonstrations are not included in the [`oscal-compass`](https://github.com/oscal-compass) organization. Demonstrations are confined the `oscal-compass-demos` organization.
+2. Due to the nature of Oscal-compass, demonstrations are likely to span multiple repositories. Each demonstration should at least have a descriptive name which is the root repository of the demonstration that includes documentation: 
```

Comment:
To enforce this consistency, would `compliance-trestle-agile-authoring` be the creator and manager of our demo content? If this could be fully automated (perhaps this could be a future phase or stretch goal), the request for new demo content could be through a PR to a catalog of use cases under `oscal-compass-demos` and Agile Authoring could be triggered to create it on PR merge.

Reply:
I think that's a good aspirational goal. My focus was more on how do we demonstrate some different functionalities in a prescribed scenario. e.g. for Kubecon Japan @yana1205 and I have been working on a scenario that will require a user to deploy manifests to a k8s cluster.

In that scenario We'd need at least two repos - one for the component definition (to connect c2p and kyverno to a cluster) and one for the cluster manifiests / argoCD applications.

Reply:
We recently created a set of repo's for an e2e demo anchored here: [e2e-demo](https://github.com/oscal-compass/e2e-demo). The initial use was for KubeCon EU 2025 (last week).

The demo video show two aspects: 
1. compliance-as-code -> agile authoring
2. policy-as-code -> c2p and automated posture calculation of a Ubuntu VM

The user can easily run part 2 by downloading the posture repo and running `make demo`. Pre-req's are Virtual Box and Vagrant. Cloning the repo's and setting up for agile authoring for part 1 is a bit more complex.

I suggest that adding kyverno/kubernetes as another target of compliance should be done here, or we somehow merge.